modules/system/defaults/WindowManager.nix

```diff
@@ -56,6 +56,13 @@ with lib;
         Hide items in Stage Manager.
       '';
     };
+    system.defaults.WindowManager.EnableTiledWindowMargins = mkOption {
+      type = types.nullOr types.bool;
+      default = true;
+      description = ''
+        Enable Window Margins.
```

Comment:
What's the default in macOS?

Reply:
true

Reply:
Can you update the description to include `The default is true.`?